db/migrate/20221124152329_add_exercise_to_submissions.rb

```diff
@@ -0,0 +1,16 @@
+class AddExerciseToSubmissions < ActiveRecord::Migration[7.0]
+  def change
+    unless Rails.env.production?
+      add_column :submissions, :exercise_id, :mediumint, null: true
```

Comment:
We currently have 4000+ exercises. :smallint has room for 32_000. Do we want to use that or be safer with :mediumint?

Reply:
mediumint